docs/source/en/tutorials/typescript.md

```diff
@@ -127,30 +127,75 @@ export interface NewsItem {
 
 import { Context } from 'egg';
 
-export default function robotMiddleware() {
+export default function xxxMiddleware() {
```

Comment:
use meanfull word

Reply:
This is an example, so what I mean is just to make the as a cusomized name with "xxxx".

Reply:
`foo`, `bar` may be better.

Reply:
Fixed according to @killagu